docs/components/modeler/forms/form-element-library/forms-element-library-iframe.md

```diff
@@ -19,5 +19,35 @@ Every iFrame component is a sandbox. This means that the content of the iFrame i
 - **Title**: Label displayed on top of the iFrame and as the accessible title. Can either be an [expression](../../feel/language-guide/feel-expressions-introduction.md), plain text, or [templating syntax](../configuration/forms-config-templating-syntax.md).
 - **URL**: Enter an HTTPS URL to a source. Can either be an [expression](../../feel/language-guide/feel-expressions-introduction.md), plain text, or [templating syntax](../configuration/forms-config-templating-syntax.md). Ensure the URL is safe as it might impose security risks. Not all external sources can be displayed in the iFrame. Read more about it in [the X-FRAME-OPTIONS documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options).
 - **Height**: Defines the height of the iFrame. Defined as number of pixels.
+- **Security attributes**: Allow the iframe's sandbox more access to various functionality of the browser, at the cost of security.
```

Comment:
I am going to opt for "iFrame" for consistency here -- do you have any strong opinions?

Reply:
I fell down a rabbit hole with this one, but I think these sources are the truthiest: 

* https://www.w3schools.com/html/html_iframe.asp
* https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Other_embedding_technologies

Iframe (if the first word in a sentence), otherwise, iframe or `<iframe>` but there are plenty of other pages using iFrame too.

Reply:
Well consistency's gotta start somewhere so we've got iframe now :)